backend/src/test/e2e/link.test.mjs

```
@@ -22,17 +22,17 @@ const createHelper = async (token, helper) => {
+    .post('/api/helper-link/add_helper')
+    .set('Authorization', `Bearer ${token}`)
     .send({
       ...helperData,
       links: [],
     });
   return res.body;
 };
 
-describe("E2e tests link", () => {
-  describe("POST /api/link/add_link", () => {
+describe.only('E2e tests link', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)